RecFCCeeCalorimeter/src/components/AugmentClustersFCCee.cpp

```diff
@@ -67,8 +67,9 @@ StatusCode AugmentClustersFCCee::initialize()
   }
 
   // initialise the list of metadata for the clusters
-  // append to the metadata of the input clusters
-  std::vector<std::string> showerShapeDecorations = m_inShapeParameterHandle.get();
+  // append to the metadata of the input clusters (if any)
+  std::vector<std::string> emptyVector;
+  std::vector<std::string> showerShapeDecorations = m_inShapeParameterHandle.get(emptyVector);
```

Comment:
```suggestion
  const auto showerShapeDecorations = m_inShapeParameterHandle.get({});
```
should also work, I think. If that doesn't, you can also just put `std::vector<std::string>{}` into the call to `get`. Similar on the version below.